src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Common practice in java is to declare enum values in upper case, but if you want enam values to be printed in lower case consider overriding toString()